How to handle race/ethnicity categories (#1484)
ihmeuw/vivarium_research: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

2.1 Data sources
----------------

As noted above, the government provides standardized race and ethnicity 
categories, though these do change over time. Currently, the race 
categories are American Indian or Alaska Native, Asian, Black or African American, 
Native Hawaiian or Other Pacific Islander, and White. The ethnicity 
categories are Hispanic or Latino and Not Hispanic or Latino. 

By having a set standard, 
it improves the chance that multiple data sources will have the same 

> categories. However, this is not guarunteed. Here, we will go through 

Reviewer: ```suggestion
categories. However, this is not guaranteed. Here, we will go through 
```

a few scenarios of how race and ethnicity categories might be misaligned 
between datasets and the options for handling each. 

> **Nested Categories**

Reviewer: Should these be headers, so they are linkable/in the table of contents?

Author: I actually went back and forth on this myself. Are these important enough to be in the table of contents? But if you think it would be helpful happy to change.

In this guide, we use nested categories to mean that one dataset has more granular 
categories than another. For example, one dataset might report a race group 
as simply "Asian" whereas another might have many categories that fit within 

> this group like Chinese, Filipino, Japanese, or Korean. In fact, the US Census 

Reviewer: ```suggestion
this group like Chinese, Filipino, Japanese, or Korean. In fact, the US government
```

Maybe?

often collects both the more granular and rolled up categories of data. 

Granular data is usually better, as different groups within a single race 
category can have very different experiences and needs. Try to keep the 
most granular data possible. However, granular data can sometimes create 

> issues with statisitcal power or small sample size. Weigh the different 

Reviewer: ```suggestion
issues with statistical power or small sample size. Weigh the different 
```

pros and cons as they relate to your project, centering people's 
identities and needs in the conversation. 

**Non-Nested Categories**

If you need to combine multiple datasets, you will likely have to combine 
race and ethnicity categories that won't match up perfectly. One option 
is to take the "least common denominator" approach, which essentially means 

> use the most granular race and ethnicity categories possible that still 
> captures the data fully. 

Reviewer: ```suggestion
using the most granular race and ethnicity categories possible that still 
capture the data fully. 
```

Often this will end up being the US standard categories above. We strongly 
recommend against using fewer categories than the US standard unless it 
is impossible to do otherwise. Creating larger buckets ends up merging 
individuals with diverse experiences, backgrounds, and identities. Consider 
what there is to be gained from this analysis and if race and ethnicity are 
important to include if you plan to use fewer categories than the US standard. 

Another option is to attempt a crosswalking approach if one or more of your 
input datasets use different categories than the standard. This would allow 
you to retain all of the data inputs, while keeping some granularity in categories. 

**Multiracial Groups**

Part of the US standard approach is allowing people to select as many race and 

> ethnicity categories as they identify with. Many people have multiracial 

Reviewer: I'm actually not totally sure whether this is standard or not!

Author: I think it is! Here is the link I have been using: https://www.govinfo.gov/content/pkg/FR-1997-10-30/pdf/97-28653.pdf 

And on page 5 they say "When self-identification is used, a method for reporting more than one race should be adopted. The method for respondents to report more than one race should take the form of multiple responses to a single question and not a ‘‘multiracial’’ category."

identities and capturing this is important. However, it creates a statistical 
issue without an easy answer - how do you handle overlapping groups? 

To understand the options below, let's provide an example of a single 
person who selected both "Black or African American" and "Asian" for 
their race. 

The first option is to include all 

> relevant race/ethnicity groups. So for this example, you would have 
> groups for "Black or African American", "Asian", and "Both Black or African 
> American and Asian". If your dataset is large 

Reviewer: ```suggestion
combinations of race/ethnicity groups. So for this example, you would have 
groups for "Black or African American alone", "Asian alone", and "Both Black or African 
American and Asian". If your dataset is large 
```

Just an idea -- I think these "alone" terms are commonly used in government data.

enough to support having this level of granularity in groups, this 
approach can work well. However, often this leads to issues with 
small sample sizes. 

The second option is to exclude everyone who selected multiple race 
or ethnicity groups. So we would just not count our example individual. We do not 
recommend this approach. 

The third is to create a large, "multiracial" group. The resulting 
groups would then be "Black or African American", "Asian", and "Multiracial". We also do 
not recommend this approach as the resulting group is generally 
too diverse to have any meaningful conclusions about.

The fourth is to count people in all race/ethnicity groups they 
selected. Therefore in our example, the individual would be 
counted twice - once in "Black or African American" and once in "Asian". This 
can lead to double counting in the data which might be more or 
less important depending on the size of the multiracial group 

> in the dataset. This might be a reasonable option.

Reviewer: ```suggestion
in the dataset and the type of analysis. This might be a reasonable option.
```

multiracial group. For example, you would have categories of "Asian" 
and "Asian alone" both existing. "Asian" would include anyone who 
selected Asian, including the multiracial person in our example, 
and "Asian alone" would be people who only identify as 
Asian, excluding the example individual. Often people will present 
anlyses for both of these groups. This is also a reasonable 
option. 

Last, you can attempt to crosswalk individuals into a single 
race/ethnicity group. There have been multiple attempts to do 
this based on studies that allow respondents to select all racial 
categories they identify with and then to pick a single one they 
most identify with. [Liebler_2008]_ Therefore, the example individual 
would be placed in a single racial group - either "Black or African 
American" or "Asian" based on their other data. If this is 
feasible based on the data present, it is also a reasonable approach. 

Often, someone will have made the decision about how to handle 
race/ethnicity categories and multiracial individuals before you 
receive the data. In this case, try to find out what was assumed 
and note it appropriately in your limitations as needed. There is 

> more information on this in the section below. 

Reviewer: I feel like this last paragraph could be made more prominent, since this is almost always the situation we are in. Maybe we should say this first, then say "there are a number of different ways your data might code race and ethnicity" and list the options you have above?
